### ISSUE TITLE
Air Strafe sluggish

### ISSUE THREAD

**Issue by TheKnightofAura** [naelstrof/KoboldKare#239]
Latest steam update seems to have made the acceleration from Bhop and airstrafe to be significantly lower, making it harder to build up initial speed

**Comment by Shurutsue** [comment]
Probably the result of https://github.com/naelstrof/KoboldKare/commit/900181c9919d24983f1dd178f3ca524d1757e64e
> **Nerfed ability to slide up unwalkable surfaces to be similar to source.**
> This also slightly nerfs bhop acceleration, but not by much.

**Comment by TheKnightofAura** [comment]
Ah, that does look like the cause. I figured it was intentional- but I was playing with a friend earlier, beating the surf map, and we both agreed that it doesn't feel nearly as good as it did before to traverse around. Consider this my leaving an opinion on the matter

**Comment by naelstrof** [comment]
Yeah thanks for leaving your opinion! I wonder how I can address it better, it does indeed feel worse, but before I was very easily able to climb up 89 degree surfaces just by holding w on them because the air acceleration was so high. I plan to make it adjustable via scene descriptor, which will allow map creators to set a custom air acceleration cap.

I'll try to update this issue when I do.
